BUILD.gn

```diff
@@ -194,6 +223,7 @@ static_library("pugixml") {
     "-Wno-unused-private-field",
     "-Wno-invalid-partial-specialization",
     "-Wno-array-bounds",
+    "-Wno-implicit-fallthrough",
```

Comment:
After rebase, the implicit fallthrough messages should have been resolved.. Can you check again by removing this flag?